satsense/image.py

```diff
@@ -61,6 +65,21 @@ def gray_ubyte(self):
             self._gray_ubyte_image = get_gray_ubyte_image(self.rgb, RGB)
         return self._gray_ubyte_image
 
+    @property
+    def canny_edged(self):
+        try:
+            if self._canny_edge_image is None:
+                grayscale = np.copy(self.grayscale)
```

Comment:
Why the copy? It looks like [`skimage.filters.rank.equalize`](http://scikit-image.org/docs/dev/api/skimage.filters.rank.html#skimage.filters.rank.equalize) creates a new output array, the way you use it.

Reply:
Hmm I guess this is leftover from debugging